public/app/features/transformers/editors/HistogramTransformerEditor.tsx

```diff
@@ -51,7 +51,13 @@ export const HistogramTransformerEditor: React.FC<TransformerUIProps<HistogramTr
           label={histogramFieldInfo.bucketSize.name}
           tooltip={histogramFieldInfo.bucketSize.description}
         >
-          <Input type="number" value={options.bucketSize} placeholder="auto" onChange={onBucketSizeChanged} min={0} />
+          <Input
+            type="number"
+            value={options.bucketSize ?? ''}
+            placeholder="auto"
+            onChange={onBucketSizeChanged}
+            min={0}
+          />
```

Comment:
When user is trying to enter in fractional bucket size (i.e. 0.5) the `0` doesn't appear which may be a confusing to end user. User can enter in `.5` just fine, but for some reason entering in `0` never appears. User can type all of `0.5` to achieve desired bucket size, but UX of this is less than ideal